kpis/kpi_calculator.py

```diff
@@ -630,6 +666,78 @@ def get_computational_time_ratio(self):
 
         return time_rat
 
+    def get_actuator_travel(self):
+        '''This method returns the measure of the actuator travel displacement when accounting for the sum of all
+        the actuators present in the test case.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        ltra_tot: float
+            displacement of actuator travel
+
+        '''
+        self.ltra_tot = 0.
```

Comment:
Add space between end of docstring and start of code (consider where needed elsewhere too).  A code format I'd like to keep consistent.